packages/node_modules/@webex/plugin-devicemanager/README.md

```diff
@@ -22,31 +22,31 @@ npm install --save @webex/plugin-devicemanager
 
 This is a plugin for the Cisco Webex JS SDK . Please see our [developer portal](https://developer.webex.com/sdks-and-widgets.html) and the [API docs](https://webex.github.io/spark-js-sdk/api/) for full details.
 
-Samples
-// refer @webex/samples/browser-devicemanager/app.js
+###Samples
+
+```
+npm run samples:devicemanager
+```
 
 // initialize spark instance -> connects to mercury -> registers device ->
 // initializes plugin-devicemanager to listen for device updates
 
-``` javascript
-import '@webex/plugin-devicemanager';
-import SparkCore from '@ciscospark/spark-core';
+```javascript
+import "@webex/plugin-devicemanager";
```

Comment:
The SDK has a lint rule for single quotes. Not that this won't work for external code, but if people are contributing back and see this as an example, it could cause some confusion.

Reply:
yeah should definitely be single. Let me look at that